igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsProperties.groovy

```diff
@@ -45,6 +45,8 @@ class JenkinsProperties {
         @NotEmpty
         String password
 
+        Boolean preventCSRF = false
```

Comment:
can we just change this to `csrf` and have it be true when it is enabled and false when not? `Cross-Site Request Forgery = true / false` is more user friendly.